mountpoint-s3/src/fs.rs

```diff
@@ -528,6 +528,7 @@ where
 {
     pub async fn init(&self, config: &mut KernelConfig) -> Result<(), libc::c_int> {
         let _ = config.add_capabilities(fuser::consts::FUSE_DO_READDIRPLUS);
+        let _ = config.add_capabilities(fuser::consts::FUSE_ATOMIC_O_TRUNC);
```

Comment:
This one we shouldn't ignore the return value — if the host doesn't support `FUSE_ATOMIC_O_TRUNC` we should fail, or disable overwrites? I don't know off the top of my head if this works on macOS, for example.